src/legacy/ui/public/vis/editors/default/default.html

```diff
@@ -11,6 +11,7 @@
 
   <div
     class="visEditor__canvas"
+    data-shared-items-container
```

Comment:
@tsullivan, I wonder if we need data-title, data-description here? Them are added to a single nested `data-shared-item`, but I wonder if it also should be added here?

Reply:
`data-title`, `data-description` are not needed here. :)